projects/pTokens/index.js

```diff
@@ -0,0 +1,81 @@
+const sdk = require("@defillama/sdk");
+const utils = require("../helper/utils");
+const { toUSDTBalances } = require("../helper/balances");
+
+const API_URL = "https://chart.ptokens.io/index.php?a=assets";
+
+async function calcTvl(CHAIN) {
+  let totalLiquidityUSD = 0;
+
+  (await utils.fetchURL(API_URL)).data
```

Comment:
would be great if this worked with historical data points too

Reply:
Yes, I saw the one already is done has the historical data. Then we can left that one and don't do the overwrite, I think!